src/pages/ProductDetail.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import Button from '../components/Button/Button';
+import styles from './ProductDetail.module.css';
+
+export default function ProductDetail() {
+  const {
+    state: { id, title, img, price },
+  } = useLocation();
+  const handleClick = (e) => {
```

Comment:
```suggestion
  const handleClick = (event) => {
```
e대신에 event 로 축약어는 피해주세요 
https://www.notion.so/hnhyerim/code-updated-12-4-f4f8aa0f6de9424ab2cad142df625d83#3a9a7b61ea1544f4b1ff6624c40e2894

Reply:
넵 주의하겠습니다!